includes/compat/trait-wc-stripe-subscriptions.php

```
@@ -373,6 +373,41 @@ public function scheduled_subscription_payment( $amount_to_charge, $renewal_orde
+		if ( ! empty( $response->error->charge ) && is_string( $response->error->charge ) ) {
+			$charge_id = $response->error->charge;
+		} elseif ( ! empty( $response->error->payment_intent->latest_charge ) && is_string( $response->error->payment_intent->latest_charge ) ) {
+			// Fallback: Payment Intents API may surface the charge ID via latest_charge.
+			$charge_id = $response->error->payment_intent->latest_charge;
+		}
+
+		if ( empty( $charge_id ) ) {
+			return false;
+		}
+
+		$charge = WC_Stripe_API::retrieve( "charges/{$charge_id}" );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_API' in method 'is_charge_blocked_by_radar'. (undefined)

(StaticAccess)